fedimint-client/src/sm/executor.rs

```diff
@@ -775,6 +830,16 @@ where
     type Record = InactiveStateKey<GC>;
 }
 
+enum ActiveOrInactiveState<GC> {
```

Comment:
Kind of a weird name. `TriggerFunctionState` (or something? I'm not sure yet what it is)

Reply:
`StateTransitionOutcome` maybe? Although I don't have a problem with the current name.